patterns/behavioral/chain_of_responsibility.py

```diff
@@ -19,11 +19,10 @@
 """
 
 from abc import ABC, abstractmethod
-from typing import Optional, Tuple
 
 
 class Handler(ABC):
-    def __init__(self, successor: Optional["Handler"] = None):
+    def __init__(self, successor: "Handler | None" = None):
```

Comment:
I think you meant this: `def __init__(self, successor: "Handler" | None = None):`